resource.go

```diff
@@ -125,7 +126,7 @@ func (r Resource) Tags() map[string]string {
 				t[kk] = vv
 			}
 		}
-	case "aws_instance", "linode_instance":
+	case "serverscom_dedicated_server", "aws_instance", "linode_instance":
```

Comment:
This seems to be a separate feature. Could you please split this into a separate PR?